packages/cfpb-design-system/src/components/cfpb-forms/label.scss

```diff
@@ -18,17 +18,11 @@
   }
 
   &--heading {
-    @include h4;
+    display: block;
 
-    // TODO: Move this above the h4 mixin. Mixin needs to allow margin override.
-    /* stylelint-disable no-duplicate-selectors */
-    & {
-      display: block;
+    margin-bottom: math.div(10px, $size-iv) + em;
 
-      // Overwrites heading-4 margin.
-      margin-bottom: math.div(10px, $size-iv) + em;
-    }
-    /* stylelint-enable */
+    @include heading-4($has-margin-bottom: false);
```

Comment:
Why not combine these two lines by changing the mixin API to something like `heading-4($margin: <whatever>);`? And give `$margin` a default value in the mixin definition.

Reply:
Mostly because `margin-top` has a different value across breakpoints in `heading-1` and `heading-2`. In the FIG these get overridden as a flat value across all breakpoints ([i.e.](https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/unprocessed/apps/filing-instruction-guide/css/main.scss#L14)), whereas in other cases it adjusts across breakpoints. So I didn't see a non-confusing way to handle both of those scenarios. Margin-bottom could be `$margin`, as could the margin on `heading-3` through `heading-6`, but it seemed confusing to have a different margin signature based on the margin and heading size. I don't love this setup though, so certainly open to better ways to engineer this. Maybe this can go in and it can get refined from here.